deploy/observability/dynamo_local_resource_monitor.py

```
@@ -0,0 +1,582 @@
+    pynvml.nvmlInit()
+    HAS_NVML = True
+except pynvml.NVMLError as e:
+    print(f"NVML init failed ({e}) -- GPU monitoring disabled")
+
+
+def parse_args():
+    p = argparse.ArgumentParser(
+        description="GPU/CPU/Disk/Network Prometheus metrics exporter"
+    )
+    p.add_argument("--port", type=int, default=8051)
+    p.add_argument("--host", default="0.0.0.0")
```

Ruff (0.15.12)

[error] Possible binding to all interfaces

(S104)